src/services/extensions/exposedComponents.tsx

```diff
@@ -0,0 +1,27 @@
+import { LinkButton } from '@grafana/ui';
+import { OpenInExploreLogsButtonProps } from 'Components/OpenInExploreLogsButton';
+import React, { lazy, Suspense } from 'react';
+const EntityAssertionsWidget = lazy(() => import('Components/OpenInExploreLogsButton'));
```

Comment:
We still have to include the entire `'Components/OpenInExploreLogsButton'` component in the main bundle since you load the interface from this same component. Maybe move the `OpenInExploreLogsButtonProps` into a new file?